tools/lib/esbuildConfig.js

```diff
@@ -69,16 +64,15 @@ const esbuildConfig = ({ nodeEnv, name }) => {
 		outfile,
 		bundle: true,
 		sourcemap: true,
-		jsx: "automatic",
-		jsxDev: nodeEnv === "development",
+		inject: ["tools/lib/react-shim.mjs"],
```

Comment:
This file doesn't exist anymore. Maybe you were basing this on an old version of esbuildConfig.js?

Reply:
100% could be the case. I think there is something wrong with my fork. I forked the repo way back in 2019, I am sure lots of changes have been made since.